src/screens/Home/components/SideContent/index.js

```diff
@@ -0,0 +1,145 @@
+  });
+  const [country, setCountry] = useState();
+
+  return loading || !country ? (
+    <Loading isSmall />
+  ) : (
+    <div className={styles.content}>
+      <img
+        className={`m-bottom-3 ${styles.img}`}
+        src={country.flag.svgFile}
+        alt={country.name}
+      />
+      <h2 id='dialog-title' className='small-title m-bottom-1'>
+        {country.name}
+      </h2>
+      <h3 className='big-text fw-thin m-bottom-3'>
+        <span
+          className='base-text m-right-1'
+          role='img'
+          aria-label={country.name}
+        >
+          {country.flag.emoji}
+        </span>
+        {country.nativeName} ({country.alpha3Code})
+      </h3>
+      <div className={`m-bottom-1 ${styles.innerContent}`}>
+        <div className={`m-bottom-2 m-right-1 ${styles.group}`}>
+          <span className='small-text m-bottom-1'>
+            {i18next.t('HOME:COUNTRY_AREA')}
+          </span>
+          <span className='base-text'>{country.area}</span>
+        </div>
+        <div className={`m-bottom-2 m-right-1 ${styles.group}`}>
+          <span className='small-text m-bottom-1'>
+            {i18next.t('HOME:COUNTRY_POPULATION')}
+          </span>
+          <span className='base-text'>
+            {currencyFormat(country.population)}
+          </span>
+        </div>
+        <div className={`m-bottom-2 ${styles.group}`}>
+          <span className='small-text m-bottom-1'>
+            {i18next.t('HOME:COUNTRY_CAPITAL')}
+          </span>
+          <span className='base-text'>{country.capital}</span>
+        </div>
+      </div>
+      <div className={`m-bottom-1 ${styles.innerContent}`}>
+        <div className={`m-bottom-2 m-right-1 ${styles.group}`}>
+          <span className='small-text m-bottom-1'>
+            {i18next.t('HOME:COUNTRY_DEMONYM')}
+          </span>
+          <span className='base-text'>{country.demonym}</span>
+        </div>
+        <div className={`m-bottom-2 m-right-1 ${styles.group}`}>
+          <span className='small-text m-bottom-1'>
+            {i18next.t('HOME:COUNTRY_NUMERIC_CODE')}
+          </span>
+          <span className='base-text'>{country.numericCode}</span>
+        </div>
+        <div className={`m-bottom-2 ${styles.group}`}>
+          <span className='small-text m-bottom-1'>
+            {i18next.t('HOME:COUNTRY_CALLING_CODES')}
+          </span>
+          <ul className={`base-text ${styles.list}`}>
+            {country.callingCodes.map((el) => (
+              <li key={el._id}>{el.name}</li>
+            ))}
+          </ul>
+        </div>
+      </div>
+      <div className={`m-bottom-3 ${styles.group}`}>
+        <span className='small-text m-bottom-1'>
+          {i18next.t('HOME:COUNTRY_SUBREGION')}
+        </span>
+        <span className='base-text'>
+          {country.subregion.name} - {country.subregion.region.name}
+        </span>
+      </div>
+      <div className={`m-bottom-3 ${styles.group}`}>
```

Comment:
here ass well, this could be another component since you're repeating this pattern 3 times